lib/handlers/injector-sor.ts

```diff
@@ -569,7 +570,7 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
     } catch (err) {
       switch (protocol) {
         case Protocol.V4:
-          return new StaticV4SubgraphProvider(chainId, poolProvider as IV4PoolProvider)
+          return new StaticV4SubgraphProvider(chainId, poolProvider as IV4PoolProvider, v4PoolsParams)
```

Comment:
forgot to pass v4PoolsParams to static subgraph provider in the prior [PR](https://github.com/Uniswap/routing-api/pull/872/files#diff-42941f26ebe3855c458055f7f2269f22a2f85265c24cb1c5ac8dd78d20e0a947R487). also fixing it here.